plugin/src/main/java/com/denizenscript/denizen/events/block/BlockExplodesScriptEvent.java

```diff
@@ -41,6 +40,26 @@ public class BlockExplodesScriptEvent extends BukkitScriptEvent implements Liste
 
     public BlockExplodesScriptEvent() {
         registerCouldMatcher("<block> explodes");
+        this.<BlockExplodesScriptEvent, ListTag>registerOptionalDetermination(null, ListTag.class, (evt, context, value) -> {

```

Comment:
Have you tested this event's determines thoroughly? ListTag vs ElementTag input is... iffy. eg I can't think of any reason this code would reliably know that `- determine <player.location>` should go to List and not Element